src/main/dig/lib/DIL Chips/74xx/counter/74162.dig

```diff
@@ -1,18 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <circuit>
-  <version>1</version>
+  <version>2</version>
   <attributes>
     <entry>
       <string>shapeType</string>
       <shapeType>DIL</shapeType>
     </entry>
     <entry>
       <string>Description</string>
-      <string>decimal synchronous counter</string>
+      <string>Decimal Synchronous Counter</string>
     </entry>
     <entry>
-      <string>lockedMode</string>
-      <boolean>true</boolean>
```

Comment:
Library components have to be locked, otherwise the build will fail.

Reply:
I didn't make this change intentionally. It happened behind the scenes when I made the more obvious edits. I don't know how to undo it or keep it from happening again.

Reply:
In order to be able to edit the circuit at all, the locking must first be switched off in the circuit-specific settings. Once editing is complete, the lock must then be switched back on. This lock is enforced in the library components during the build to prevent users from accidentally changing the library components.